pkg/api-server/pagination.go

```
@@ -40,16 +38,21 @@ func pagination(request *restful.Request) (page, error) {
 	}, nil
 }
 
-func nextLink(request *restful.Request, publicURL string, list model.ResourceList) (*string, error) {
+func nextLink(request *restful.Request, list model.ResourceList) (*string, error) {
```

Seems like we don't need to return an error in this method